pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancerClusterStatelessTest.java

```diff
@@ -1331,10 +1303,8 @@ public void testRebalancePreCheckerRebalanceConfig()
   }
 
   /**
-   * Tests rebalance with tier configs
-   * Add 10 segments, with segment metadata end time 3 days apart starting from now to 30 days ago
-   * 1. run rebalance - should see no change
-   * 2. add nodes for tiers and run rebalance - should see no change
+   * Tests rebalance with tier configs Add 10 segments, with segment metadata end time 3 days apart starting from now to
+   * 30 days ago 1. run rebalance - should see no change 2. add nodes for tiers and run rebalance - should see no change
```

Comment:
nit: the earlier formatting was somewhat easier to read, is this change intentional?

Reply:
Good catch. Didn't realize my IDE is formatting the javadoc